mat/eigen_test.go

```diff
@@ -153,25 +154,26 @@ func TestSymEigen(t *testing.T) {
 		var es EigenSym
 		ok := es.Factorize(test.mat, true)
 		if !ok {
-			t.Errorf("bad factorization")
+			t.Errorf("bad test")
```

Comment:
These might want some identifying mark to explain which case was bad — pointless here with only one case, but below might be good. Not crucial.

Reply:
Done.